components/02-molecules/accordion/accordion.twig

```diff
@@ -0,0 +1,37 @@
+{% set accordion__base_class = 'accordion' %}
+
+<div {{ bem(accordion__base_class) }} data-component-width="content">
+  <div {{ bem('inner', [], accordion__base_class) }}>
+    {% include "@atoms/typography/headings/heading.twig" with {
+      heading__level: '3',
+      heading__blockname: accordion__base_class,
+      heading: accordion__heading,
+    } %}
+    {% embed "@atoms/lists/list.twig" with {
+      list__base_class: 'controls',
+      list__blockname: accordion__base_class,
+      list__attributes: {'aria-labels': 'Section controls'},
```

Comment:
Change to aria-label

Reply:
I was thinking, can we have ARIA attributes added to our linter? I have misspelled aria 1000x (usually aria-labelledby, lol).

Reply:
Done.

I don't know of any way to check for mis-spellings... If you find something though, it would be worth looking into. (My brief google search didn't find anything)